visualizer/semanticsHelper.js

```diff
@@ -253,7 +277,7 @@
       if (operationName in semantics._getSemantics().operations) {
         var argValues;
         if (operationName === opName) {
-          argValues = toValueList(opArguments || Object.create(null));
+          argValues = toValueList(optArguments || opArguments || Object.create(null));
```

Comment:
Consider renaming one of these variables -- `opArguments` and `optArguments` are *really* similar, and it's easy to get confused.